phone/src/locale/no.json

```
@@ -38,28 +38,28 @@
         "FRAME": "Deksel",
         "ZOOM": "Zoom",
         "WALLPAPER": "Bakgrunn",
-        "RESET_SETTINGS": "Fabrikkinstill",
-        "RESET_SETTINGS_DESC": "Dette fjerner all lokal data",
+        "RESET_SETTINGS": "Nullstill telefon",
+        "RESET_SETTINGS_DESC": "Dette fjerner alt innhold og alle innstillinger",
         "STREAMER_MODE": {
           "TITLE": "Strømmermodus",
           "DESCRIPTION": "Gjem bilder før de vises"
         },
         "CUSTOM_WALLPAPER": {
-          "DIALOG_TITLE": "Tilpasset Bakgrunnsbilde",
+          "DIALOG_TITLE": "Tilpasset bakgrunnsbilde",
           "DIALOG_CONTENT": "Angi et tilpasset bakgrunnsbilde ved å skrive inn bilde url-en nedenfor!",
           "DIALOG_PLACEHOLDER": "Bilde URL",
           "DIALOG_ERROR": "Ugyldig bilde eller link"
         },
-        "CALL_VOLUME": "Call Volume",
+        "CALL_VOLUME": "Samtalevolum",
         "RINGTONE": "Ringetone",
-        "RINGTONE_VOLUME": "Ringetone Volum",
-        "NOTIFICATION": "Varsling",
+        "RINGTONE_VOLUME": "Ringelyd",
+        "NOTIFICATION": "Varsellyd",
         "NOTIFICATION_VOLUME": "Varslingsvolum",
-        "NOTIFICATION_FILTER": "Varslingsfilter"
+        "NOTIFICATION_FILTER": "Varsle meg om"
       },
       "MESSAGES": {
-        "INVALID_SETTINGS": "Ugyldige lagret innstillinger, vennligst resett innstillinger.",
-        "SETTINGS_RESET": "Innstillingene dine har blitt resatt!"
+        "INVALID_SETTINGS": "Ugyldig lagrede innstillinger, vennligst resett innstillinger og prøv igjen",
+        "SETTINGS_RESET": "Innstillingene er nå resatt!"
```

```suggestion
        "SETTINGS_RESET": "Innstillingene er nå tilbakestilt!"
```
This might be better.

---

> This might be better.

Yep, I agree! 👍🏼